lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -248,6 +248,35 @@ function dosomething_northstar_save_id_field($uid, $northstar_response) {
 }
 
 /**
+ * Save the user's Northstar ID to their local profile.
+ *
+ * @param object $account
+ *   Fully loaded user object.
+ *
+ * @return string
+ *   The Northstar id or empty string on error.
+ */
+function dosomething_northstar_get_id($account = NULL) {
```

Comment:
hmm, then perhaps that too belongs in the `dosomething_user_get_field` block.
in an effort to simplify the code, it would be really nice to have one way to call field info on a user, everywhere instead of breaking out another one off solution here.